.circleci/config.yml

```diff
@@ -1,103 +1,33 @@
 ---
-version: 2.1
+# Prometheus has switched to GitHub action.
```

Comment:
Let me know if we should just delete this file entirely. I think we need it short term while we get PRs merged or rebased.